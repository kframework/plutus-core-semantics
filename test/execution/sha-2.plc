(program

(module Foo (import) (export () ())
  (define result (builtin sha2_256 #ABCD)))

)
