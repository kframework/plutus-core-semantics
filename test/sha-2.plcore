(program

(module Foo (imported) (exported () ()) (local () ())
  (define result (builtin sha2_256 #ABCD)))

)
