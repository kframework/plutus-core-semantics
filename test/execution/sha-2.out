<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . result ) |-> #e12e115acf4552b2568b55e93cbd39394c4ef81c82447fafc997882a02d23677 </genv> <mod> Foo </mod> </T>
